libopenage/util/compiler.h

```diff
@@ -8,18 +8,44 @@
  *
  * May contain some platform-dependent code.
  */
+ #include <ciso646>
 
 // pxd: from libcpp.string cimport string
 #include <string>
 
+/**
+ * DLL entry-point decorations.
+ */
+#if defined(_WIN32)
+	#if defined(libopenage_EXPORTS)
+		#define LIBOPENAGE_API __declspec(dllexport)
+	#else
+		#define LIBOPENAGE_API __declspec(dllimport)
+	#endif /* libopenage_EXPORTS */
+#else
+	#define LIBOPENAGE_API
+#endif
```

Comment:
Apparently it isn't. I don't think there's a large enough convention to follow out there. I used what I knew from experience. I'm certainly open to using `OAAPI` as that strikes a nice balance between a larger and a descriptive name. `OAFUNC`, not so much, because this macro is used for classes and functions alike (actually even for `extern`ed objects, but I don't know what common name all of them can use). If you're okay with `OAAPI`, I can just do a find-replace all.